content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Three Common Tenancy Models For Kubernetes'
+date: 2021-03-30
```

Comment:
We'll need a new date
/hold
until a future date is picked.

Reply:
I tentatively changed to Apr 13th but can update once we confirm a date